### PR TITLE
MG-74 - Update metadata styling for the inner tables

### LINKED ISSUE #74: [BUG] Inner tables metadata field not updated
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What were you trying to achieve?

View the tables for pages such as thing-channels, thing-users, e.t.c

### What are the expected results?

The metadata field should be updated to the latest changes. They should look as below: 
image

### What are the received results?

The metadata field of the inner tables was not updated, it is still as it was before.
image

### Steps To Reproduce

1. log into magistrala
2. enter a domain
3. click on thing
4. click on connect

### In what environment did you encounter the issue?

Magistrala UI

### Additional information you deem important

_No response_

### PR DESCRIPTION
**Pull request by ianmuchyri**
# What type of PR is this?

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Dependency Update

## What does this do?
Update the styling of the inner tables that had not been updated to the latest changes

## Which issue(s) does this PR fix/relate to?

- Resolves #74 

## Have you included tests for your changes?

- [ ] Yes
- [x] No, and this is why: 

## Did you document any new/modified functionality?

- [ ] Yes
- [x] No, and this is why:

### CODE REVIEW

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Have a look at this edge case 

image


You can test using this json structure

```json
 {
    "_id": "659e5cb57a0419df0286e45e",
    "index": 0,
    "guid": "d5eca234-7f62-4f99-8d45-7718c818d1ba",
    "isActive": true,
    "balance": "$3,406.01",
    "picture": "http://placehold.it/32x32",
    "age": 40,
    "eyeColor": "brown",
    "name": "Augusta Morales",
    "gender": "female",
    "company": "FROSNEX",
    "email": "augustamorales@frosnex.com",
    "phone": "+1 (932) 525-3174",
    "address": "779 Ocean Court, Fedora, Maryland, 2180",
    "about": "Ut do officia ipsum tempor deserunt laborum dolor irure in. Eiusmod reprehenderit duis sint incididunt dolore dolor irure exercitation nostrud. Enim aliqua nulla ut adipisicing qui. Exercitation culpa sit amet veniam mollit velit cillum amet anim veniam eu deserunt. Consequat anim nostrud duis sunt est sit velit pariatur eu nostrud enim aliqua.\r\n",
    "registered": "2019-11-07T07:28:58 -03:00",
    "latitude": 76.180218,
    "longitude": 138.835929,
    "tags": [
      "dolore",
      "adipisicing",
      "minim",
      "do",
      "ad",
      "fugiat",
      "excepteur"
    ],
    "friends": [
      {
        "id": 0,
        "name": "Mercado Ramirez"
      },
      {
        "id": 1,
        "name": "Kayla Green"
      },
      {
        "id": 2,
        "name": "Gamble Lowe"
      }
    ],
    "greeting": "Hello, Augusta Morales! You have 10 unread messages.",
    "favoriteFruit": "strawberry"
  }
```

The text is overflowing can we have max-width set to truncate long key-value pairs

File: ui/web/static/css/styles.css [hunk 1]
```diff
@@ -727,4 +731,4 @@ button.edit-btn {
 	body .body-button {
 		margin-left: 0.2rem;
 	}
-}
+}
```
Comment (rodneyosodo) [inline ui/web/static/css/styles.css:734]: new line

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
What about domain metadata? It fails to update it

**Comment by ianmuchyri** [comment]
> What about domain metadata? It fails to update it

Also a problem with Magistrala endpoint, One cannot update anything for a domain

**Review by rodneyosodo — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]
